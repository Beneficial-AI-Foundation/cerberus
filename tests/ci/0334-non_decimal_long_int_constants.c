int main(void)
{
  int ret = 0;
  // all three large constants should have type: signed long
  ret |= (-1 < 8589934591) << 2;
  ret |= (-1 < 0x1FFFFFFFF) << 1;
  ret |= -1 < 0b111111111111111111111111111111111;
  return ret; // should return 7 (0b111)
}