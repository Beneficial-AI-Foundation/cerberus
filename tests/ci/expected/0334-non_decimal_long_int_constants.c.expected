Defined {value: "Specified(7)", stdout: "", stderr: "", blocked: "false"}
